pennylane/grouping/group_observables.py

```diff
@@ -217,15 +197,29 @@ def group_observables(observables, coefficients=None, grouping_type="qwc", metho
             can be 'lf' (Largest First) or 'rlf' (Recursive Largest First)
```

Comment:
I can't comment on all cases, but string options for parameters should be written

```suggestion
            can be ``'lf'`` (Largest First) or ``'rlf'`` (Recursive Largest First)
```